2023-02-21 12:03:55,503
 - 'root' - ERROR - 19-Batch registrations: 0
20-Batch registrations: 2
21-Batch registrations: 0
22-Batch registrations: 0
23-Batch registrations: 0
24-Batch registrations: 0
25-Batch registrations: 0
'NoneType' object is not iterable
================================================================
Traceback (most recent call last):
  File "e:\RESTELE BOT\main.py", line 132, in <module>
    bot.polling(non_stop=True, timeout=30)
  File "C:\Users\gpran\AppData\Local\Programs\Python\Python311\Lib\site-packages\telebot\__init__.py", line 1043, in polling
    self.__threaded_polling(non_stop=non_stop, interval=interval, timeout=timeout, long_polling_timeout=long_polling_timeout,
  File "C:\Users\gpran\AppData\Local\Programs\Python\Python311\Lib\site-packages\telebot\__init__.py", line 1118, in __threaded_polling
    raise e
  File "C:\Users\gpran\AppData\Local\Programs\Python\Python311\Lib\site-packages\telebot\__init__.py", line 1074, in __threaded_polling
    self.worker_pool.raise_exceptions()
  File "C:\Users\gpran\AppData\Local\Programs\Python\Python311\Lib\site-packages\telebot\util.py", line 156, in raise_exceptions
    raise self.exception_info
  File "C:\Users\gpran\AppData\Local\Programs\Python\Python311\Lib\site-packages\telebot\util.py", line 100, in run
    task(*args, **kwargs)
  File "e:\RESTELE BOT\main.py", line 102, in user_details
    bot.send_message(message.chat.id, str(reg.user_details(message.text)))
                                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "e:\RESTELE BOT\register.py", line 104, in user_details
    details = dict(student_registrations[int(year)].find_one({"ROLL_NUM":roll_num}))
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: 'NoneType' object is not iterable
